test/fixture/pages/excluded-via-component-options.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+<div>
+    /excluded-via-component-options
+</div>
+</template>
+<script>
+  export default {
+    sitemap:false
```

Comment:
Why limit the `sitemap` option to a simple boolean for exclude / include a page from the sitemap?

Maybe you can support the following options in addition of `sitemap: true` :

```vue
<script>
  export default {
    sitemap: {
      changefreq: 'daily',
      priority: 1,
      lastmod: new Date(),
      // ...
   }
```